java/core/src/test/org/apache/orc/TestVectorOrcFile.java

```diff
@@ -4157,9 +4157,9 @@ public void testFutureOrcFile(Version fileFormat) throws Exception {
       String m = e.getMessage();
       assertTrue(m.contains("version1999.orc was written by a future ORC version 19.99."));
       assertTrue(m.contains("This file is not readable by this version of ORC."));
-      assertTrue(m.contains("Postscript: footerLength: 19 compression: NONE " +
-          "compressionBlockSize: 65536 version: 19 version: 99 metadataLength: 0 " +
-          "writerVersion: 1"));
+      assertTrue(m.contains("Postscript: footer_length: 19 compression: NONE " +
```

Comment:
I assume this file should not be affected.

Reply:
Ya, the error message contains the protobuf field names.